docs/topics/db/multi-db.txt

```diff
@@ -71,6 +71,11 @@ If you attempt to access a database that you haven't defined in your
 :setting:`DATABASES` setting, Django will raise a
 ``django.db.utils.ConnectionDoesNotExist`` exception.
 
+.. note::
+
+    For information about testing and multiple databases check :ref:`multi-database
+    support <emptying-test-outbox>`.
```

Comment:
This label name looks wrong, it seems to me the `.. _emptying-test-outbox:` line is in the wrong place in `testing/tools.txt`, it should be above line 1327, "Emptying the test outbox".  Then we can add a separate label above line 1113, "Multi-database support", called e.g. `.. _multi-database-support:` and use that for the link here